File: app/src/main/java/app/fynnjason/java_kotlin_networkrequest/java/ApiService.java
package app.fynnjason.java_kotlin_networkrequest.java;

import io.reactivex.Observable;
import retrofit2.http.GET;

/**
 * Created by FynnJason.
 * Des：Api
 */

public interface ApiService {
    @GET("all/20/1")
    Observable<BaseEnity<GankInfo>> allGank();
}

File: app/src/main/java/app/fynnjason/java_kotlin_networkrequest/kotlin/ApiService.kt
package app.fynnjason.java_kotlin_networkrequest.kotlin

import app.fynnjason.java_kotlin_networkrequest.java.base.BaseEnity
import app.fynnjason.java_kotlin_networkrequest.java.bean.GankBean
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Headers

/**
 * Created by FynnJason.
 * Des：api
 */
interface ApiService {
    @Headers("Cache-Control:public ,max-age=60")
    @GET("all/20/1")
    fun allGank(): Observable<BaseEnity<List<GankBean>>>
}